src/links/links.controller.ts

```diff
@@ -8,9 +8,12 @@ import {
   Param,
   Patch,
   Post,
+  ValidationPipe,
 } from '@nestjs/common';
 import { LinksService } from './links.service';
 import { Link, Prisma } from '@prisma/client';
+import { CreateLinkDto } from './dto/create-link.dto';
+import { UpdateLinkDto } from './dto/update-link.dto';
```

Comment:
**praise**: Implementing `ValidationPipe` for `createLinkDto` ensures that incoming data is validated against the DTO.